geotrek/zoning/tests/test_parsers.py

```diff
@@ -59,3 +61,100 @@ def test_multipolygon(self):
         geom = MultiPolygon(Polygon(((0, 0), (0, 1), (1, 1), (1, 0), (0, 0))))
         self.assertEqual(self.parser.filter_geom("geom", geom), geom)
         self.assertFalse(self.parser.warnings)
+
+
+class TestDistrictOpenStreetMapParser(OpenStreetMapDistrictParser):
+    provider = "OpenStreetMap"
+    tags = [[{"boundary": "administrative"}, {"admin_level": "6"}]]
+
+
+class OpenStreetMapDistrictParser(TestCase):
```

Comment:
`OpenStreetMapDistrictParser` would be the same name as the parser that's imported (line 10)

```suggestion
class OpenStreetMapDistrictParserTests(TestCase):
```